src/cpp/src/prompt_lookup/prompt_lookup_impl.cpp

```diff
@@ -84,10 +84,12 @@ void ContinuousBatchingPipeline::PromptLookupImpl::step() {
         raw_perf_counters.m_new_token_times.emplace_back(main_timer.get_end_time());
     }
 
-    if (generated_len_after.empty() && 0) {
+#ifdef LOG_LEVEL_DEBUG
```

Comment:
Can you mention somewhere this feature? I guess `SpeculativeDecodingMetrics`'s docstrings is a good place, but maybe you can come up with a better one

Reply:
There's DEBUG_LOG.md with `OPENVINO_LOG_LEVEL` so it's worth adding the description there as well

Reply:
updated, I thought that it was connected to `LOG_LEVEL_DEBUG` only